modules/admin-ui-picard/app/src/components/events/partials/modals/StartTaskModal.js

```diff
@@ -50,7 +50,16 @@ const StartTaskModal = ({ close, postTasks }) => {
                 </header>
 
                 {/* Stepper that shows each step of wizard as header */}
-                <WizardStepper steps={steps} page={page}/>
+                <WizardStepper steps={steps}
+                               page={page}
+                               setPage={nextPage}
+                               completed={false}
+                               setCompleted={() => {}}
+                               formik={{
+                                   values: {acls: []},
+                                   isValid: false,
+                                   dirty: false
+                               }}/>
```

Comment:
Not working. Take a look at e.g. NewSeriesWizard.js. There you can see how to integrate the WizardStepper correctly